astropy/cosmology/tests/test_core.py

```diff
@@ -29,7 +29,7 @@
 # SETUP / TEARDOWN
 
 
-def make_valid_zs(max_z: float = 1e5):
+def make_valid_zs(max_z: float = 1e5) -> tuple[list, np.ndarray, list]:
```

Comment:
`np.ndarray` accepts 2 type parameters. Or `NDArray` from `numpy.typing` takes only the value type

Reply:
And additionally I had missed one `list` from the annotation.